### DISCUSSION TITLE
Surround selection with tags

### DISCUSSION THREAD

**Discussion by martinbannister** [phcode-dev/phoenix#1450] [Ideas]
It would be really handy if you could highlight some text and when you type less than `<`  instead of overwriting the text a tag is started to be created either side, so you'd have `<|>the text</|>` (pipes used to represent cursors).  I often end up writing some text and then deciding I need it to be an `h1` or paragraph after the fact, or even adding a `span` to some existing text.  The current autocomplete feature is great but it means your tags are created before the text you want in them and then you have to move the existing text to inside the tags, it's a minor inconvenience, and slightly worse on large blocks of text, but could be streamlined as _highlight - type - done_.

**Comment by abose** [comment]
Thanks for the feature request. Let me see if this can be added.
